xmodule/modulestore/mixed.py

```diff
@@ -670,6 +678,14 @@ def create_course(self, org, course, run, user_id, **kwargs):  # lint-amnesty, p
         # add new course to the mapping
         self.mappings[course_key] = store
 
+        # .. event_implemented_name: COURSE_CREATED
+        COURSE_CREATED.send_event(
+            time=datetime.now(timezone.utc),
```

Comment:
Is there a relevant datetime stored on the course that we can use here instead? It won't matter much, but it would be nice if these timestamps matched.

Same question for all of these new event triggers.

Reply:
Thank you for pointing out @pomegranited. I will check this tomorrow.

Reply:
@pomegranited The course has a lot of dates, and I didn't find any more relevant than others. 

Also from [OEP-41](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0041-arch-async-server-event-messaging.html#id6):

> Timestamp that the event occurred, in UTC using [RFC 3339](https://tools.ietf.org/html/rfc3339). If this event was sent because we created a new row in the database, we should pull this time directly from the created_at field in that model so that the output matches exactly. Likewise, use updated_at for database updates. Do not call datetime.now() in these situations because we will get times that are just a few milliseconds offset from the database record of these actions. Do call datetime.now() if the event happens and has no corresponding database changes. If you are sending out multiple event messages describing the same occurance (e.g. a version 1 and version 2 of an event), they should have the exact same timestamp.

We are not using the event bus yet, so I think removing the field is better. What do you think?

Reply:
Yep, ok to remove the `time` parameter from these calls.